### ISSUE TITLE
Unknown word CssSyntaxError in Mina File

### ISSUE THREAD

**Issue by jinzhanye** [stylelint/stylelint#4709]
> Clearly describe the bug

Stylelint throws "Unknown word CssSyntaxError" in valid `.mina` files

`.mina` file  is similar to `.vue` file ，this is a simple mina file 
[https://tina.js.org/#/guide/single-file-component](https://tina.js.org/#/guide/single-file-component)

```html
<template>
  <view style="padding:{{padding}}">
      hello world
  </view>
</template>
```

the problem is, in fact, `style="padding:{{padding}}"` effect is very similar to that of 

```
:style="`padding:${padding}`"
```

both of which are dynamic settings, it's should be validated, but stylelint report it `Unknown word CssSyntaxError`

> What code is needed to reproduce the bug?

```
style="padding:{{padding}}"
```

> Which version of stylelint are you using?

13.3.1

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

with cli

```
npx stylelint '**/*.mina'
```

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, it's related to `style="padding:{{padding}}"` is not standard syntax

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

Unknown word CssSyntaxError

**Comment by jeddy3** [comment]
@jinzhanye Thanks for the report and for using the template.

You can use:

```
npx stylelint '**/*.mina' --syntax html
```

`.mina` is not an extension that stylelint recognises so it attempts to parse the files as CSS. You can use the [`syntax` option](https://stylelint.io/user-guide/usage/options#syntax) to force stylelint to parse the files as HTML. This syntax _extracts_ styles from `<style>` tags and `style=""` attributes. It seems to be compatible with the example single file component at https://tina.js.org/#/guide/single-file-component.

I'll close the issue for now. Feel free to reopen if that doesn't resolve your issue. If there are more requests for supporting the `.mina` file extension, that can be handled in [postcss-html](https://github.com/gucong3000/postcss-html).
